### PR TITLE
Sync metadata with audio

### PR DESCRIPTION
**Pull request by allending**

### CODE REVIEW

File: ExampleApp/ExampleApp/AudioPlayerView.m [hunk 1]
```diff
@@ -254,6 +271,27 @@ -(void) muteButtonPressed
 	}
 }
 
+-(void) rateControlChanged
+{
+    switch (rateControl.selectedSegmentIndex) {
```
Comment (tumtumtum) [inline ExampleApp/ExampleApp/AudioPlayerView.m]: Can you put the brace on the next line just to match the rest of the coding style ;-)

Comment (allending) [inline ExampleApp/ExampleApp/AudioPlayerView.m]: Sorry @tumtumtum. This was an unintended PR with many improper changes and inconsistent coding style. Will reopen once the changes are tested more and made consistent.

File: StreamingKit/StreamingKit/STKHTTPDataSource.m [hunk 2]
```diff
@@ -605,4 +620,142 @@ -(BOOL) supportsSeek
...
+        return 0;
+        
+    } else { // no meta data in stream
+        
+        return length;
+        
+    }
+}
+
+-(NSDictionary *) dictionaryFromMetadata:(NSString *)metaData
```
Comment (tumtumtum) [inline StreamingKit/StreamingKit/STKHTTPDataSource.m]: We may want to handle escaped characters and also handle values that are in quotes (key='value') by stripping out the quotes

**Comment by tumtumtum** [comment]
Thanks man! Just coding style issues (consistency is better than one style or another) and a small note about the metadata parsing.

**Comment by syrakozz** [comment]
+1, long time no update to this library, i hope we start get updates soon

**Pull request closed without merging**